### DISCUSSION TITLE
Enable chosen realearn instance with scene launch

### DISCUSSION THREAD

**Discussion by ElVicente** [helgoboss/helgobox#1054] [Ideas]
The idea would be to use a realearn instance to trigger changes in the session (track mute/ arm/ tempo change) using timer>0 as mapping source.
Ableton do that with automation clips I think.

**Comment by ElVicente** [comment]
Thz workflow I have in mind would is desceibed in that video.
https://m.youtube.com/watch?v=J-GeRU5SLNo&t=183s
But maybe there would be a less hacky way to do that using Playtime 2?

**Comment by helgoboss** [comment]
Automation clips are actually planned (and they will most likely leverage ReaLearn's target infrastructure): #1041

But yeah, would probably be cool if a scene could natively trigger a certain ReaLearn target.
